sdk/highlight-node/src/client.ts

```diff
@@ -184,7 +185,6 @@ export class Highlight {
 		if (metadata != undefined) {
 			span.setAttributes(metadata)
 		}
-		span.setAttributes({ ['highlight.project_id']: this._projectID })
```

Comment:
This is unnecessary to set at the span level. It's set at the resource level (see the constructor).